aur.py

```diff
@@ -15,6 +15,18 @@ def package_installed(module, package_name):
     return exit_code == 0
 
 
+def update_packages(module, tool):
+    assert tool in TOOL_TO_INSTALL_CMD_MAP
+
+    cmd = TOOL_TO_INSTALL_CMD_MAP[tool] + ['--update']
+    rc, stdout, stderr = module.run_command(cmd, check_rc=True)
+
+    module.exit_json(
+        changed=stdout.count('\n') > 2,
```

Comment:
This is really quite fragile, just set the locale to C

Reply:
ok will do, thanks for the feedback